admin_guide/backup_restore.adoc

```diff
@@ -0,0 +1,122 @@
+= Backup and Restore
+{product-author}
+{product-version}
+:data-uri:
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:description: how to backup and restore at project and cluster level
+:keywords: backup, restore
+
+toc::[]
+
+
+// REVIEWERS: READ THIS!
+//
+// In the following text, there are questions of the form:
+//   //??? QUESTION
+// Please feel free to make a line-comment to answer them, in addition
+// to any other (line-)comments on the correctness of the text.
+//
+// - Usually, the question pertains to the text preceding it.
+//   Questions pertaining to following text are explicitly noted.
+//
+// - There are bunch of questions at the end.
+//
+// Thanks for your cooperation on this (experimental) method of
+// refining the documentation.  Hopefully it will bear good fruit.
+
+== Overview
+
+In {product-title}, you can
+do _backup_, saving state to separate storage,
+and _restore_, recreating state from separate storage,
+at the cluster level.
+The full state of a cluster installation includes:
+
+- etcd data on each master
+- API objects
+- registry storage
+- volume storage
+
+This topic does not cover backup and restore of
+link:../install_config/persistent_storage/index.html[persistent storage],
+as those topics are left to the underlying storage provider.
+
+[NOTE]
+====
+To backup API objects at the project level, use `oc export`
+for each object to be saved.
+For example, to save the deployment configuration `frontend` in YAML format:
```

Comment:
I'm sorta confused at this. This is backing up API objects? Is this just another back up use case? I'm wondering if this should be explained more because atm I'm not sure why it's in a note box and not its own section.